libs/server/AOF/AofProcessor.cs

```diff
@@ -114,23 +121,23 @@ public void Dispose()
         /// <summary>
         /// Recover store using AOF
         /// </summary>
-        public unsafe void Recover(long untilAddress = -1)
+        public unsafe bool Recover(long untilAddress = -1)
         {
             logger?.LogInformation("Begin AOF recovery");
-            RecoverReplay(untilAddress);
+            return RecoverReplay(untilAddress);
         }
 
         MemoryResult<byte> output = default;
-        private unsafe void RecoverReplay(long untilAddress)
+        private unsafe bool RecoverReplay(long untilAddress)
         {
             logger?.LogInformation("Begin AOF replay");
             try
             {
                 int count = 0;
                 if (untilAddress == -1) untilAddress = storeWrapper.appendOnlyFile.TailAddress;
-                using var scan = storeWrapper.appendOnlyFile.Scan(storeWrapper.appendOnlyFile.BeginAddress, untilAddress);
+                using TsavoriteLogScanIterator scan = storeWrapper.appendOnlyFile.Scan(storeWrapper.appendOnlyFile.BeginAddress, untilAddress);
```

Comment:
Why change the vars here?

Reply:
type wasnt explicitly understood looking at the right side, figured it helps with readbility when you can tell the type from looking the right side.